SecurityService.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -31,15 +35,31 @@ public class DockerHelper
         public ISecurityServiceClient SecurityServiceClient;
         
         public Guid TestId;
-        private void SetupTestNetwork()
+        private INetworkService SetupTestNetwork()
```

Comment:
Codacy found an issue: [Change return type to 'void'; not a single caller uses the returned value.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)